csharp/src/Google.Protobuf/JsonFormatter.cs

```diff
@@ -816,6 +875,16 @@ static Settings()
             /// </summary>
             public bool FormatEnumsAsIntegers { get; }
 
+            /// <summary>
+            /// Indentation string, used for formatting.
+            /// </summary>
+            public string Indent { get; }
+
+            /// <summary>
```

Comment:
I'd suggest one of two options here:

- Remove it entirely, but convert "" to null for indentation, so that we can just use `if (settings.Indent is null)` or `if (settings.Indent is string)`
- Rename to HasIndentation or IsIndented